docs/bare-metal-promscale-stack.md

```diff
@@ -0,0 +1,185 @@
+
+This document is a guide to setup and run Promscale on bare metal (i.e a non-containerized environment) with **TimescaleDB > 2.0**.
+
+**Note**: If you prefer running your applications in containers, we recommend installing Promscale via [TOBS: The Observability Stack for Kubernetes](https://github.com/timescale/tobs) or [Docker](https://github.com/timescale/promscale/blob/master/docs/docker.md).
+
+### Overview
+
+**Target Operating System**
+The instructions below are for Ubuntu 18.04 or later (excluding obsoleted versions), but can easily be modified to deploy Promscale on the operating system of your choice. You can modify the instructions by choosing binaries of TimescaleDB, Promscale, Prometheus & Grafana which are specific to your target operating system.
+
+There are 4 steps to deploy Promscale on bare metal:
+<ol>
+<li> Deploy TimescaleDB </li>
+<li> Setup a Cronjob to compress chunks (Optional) </li>
+<li> Deploy Promscale </li>
+<li> Deploy Prometheus </li>
+</ol>
+
+Instructions to deploy and run Grafana are included as an optional, but commonly used fifth step.
+
+### 1. Deploying TimescaleDB
+
+#### Build and install TimescaleDB
+Follow the steps on timescaledb installation
+
+**If you don't already have PostgreSQL installed**, add PostgreSQL's third party repository to get the latest PostgreSQL packages (if you are using Ubuntu older than 19.04):
+```
+# `lsb_release -c -s` should return the correct codename of your OS
+echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+sudo apt-get update
+```
+
+Next, add TimescaleDB's third party repository and install TimescaleDB, which will download any dependencies it needs from the PostgreSQL repo:
+```
+# Add our PPA
+sudo add-apt-repository ppa:timescale/timescaledb-ppa
+sudo apt-get update
+
+# Now install appropriate package for PG version
+sudo apt install timescaledb-postgresql-12
+```
+#### Configure TimescaleDB
+Configure your database using `timescaledb-tune`. This will ensure that TimescaelDB is properly added to the parameter `shared_preload_libraries` as well as offer suggestions for tuning memory, parallelism, and other settings:
+```
+sudo timescaledb-tune
+
+# Restart PostgreSQL instance
+sudo service postgresql restart
+```
+
+### Access TimescaleDB
+
+To access TimescaleDB, login to your PostgresSQL database using `psql` as user `postgres`:
+```
+su - postgres
+```
+
+To open a connection to your TimescaleDB instance via psql, run:
+```
+psql
+```
+
+And finally, add TimescaleDB as an extension:
+```
+CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
+```
+
+For more instructions on installing TimescaleDB on an operating system other than Ubuntu (e.g Debian, MacOS, CentOS etc) see the [TimescaleDB install page.](https://docs.timescale.com/latest/getting-started/installation)
+
```

Comment:
We are actually missing a step here about making the promscale extension available. We should probably add it as an optional step and just compile from source as described here: https://github.com/timescale/promscale_extension